scss/components/list-group.scss

```diff
@@ -0,0 +1,81 @@
+@import "../core/settings";
+@import "../core/mixins";
+@import "../core/functions";
+
+/*!
+.tn-list-group
+  .tn-list-group__item
+    .tn-list-group__button
+    .tn-list-group__addon
+      .tn-list-group__control
+      .tn-list-group__label
+
+*/
+$block: ns(list-group);
+
+$tn-list-group--border-color: lighten(tn-color(neutral, 2), 5%);
+$tn-list-group-hover: tn-color(action);
+$tn-list-group--input-width: tn-space(7) !default;
+
+.#{$block} {
+  @include reset;
+  vertical-align: top;
+  min-height: $tn-forms-height--input-text;
+
+  &__item{
+    border: 1px solid $tn-list-group--border-color;
+    border-top: none;
+    padding: tn-space(4);
+    min-height: $tn-forms-height--input-text;
+
+    &:first-child{
+      border-top: 1px solid $tn-list-group--border-color;
+    }
+    
+    &:hover{
+      border: 1px solid $tn-list-group-hover;
+      margin-top: -1px; //enable content do not jump because of the double border
+      &:first-child{
+        border-color: $tn-list-group-hover;
+        margin-top: 0;
+      }
+    }
+    &--action{
+      position: relative;
+      padding-right: tn-space(12);
+    }
+    &--addon{
+      display: flex;
+      flex-direction: row;
+      input[type=checkbox]{
+          margin-right: tn-space(4);
+      }
+    }
+  }
+  &__control{
+    flex-basis: $tn-list-group--input-width;
+    min-width: $tn-list-group--input-width;
+  }
+  &__label{
+    flex-basis: 95%;
+  }
+  &__button{
```

Comment:
We are creating a whole new button here instead of simply extending the existing button we have. Please refer to the[ alerts component](https://github.com/SAP/techne/blob/v2.0/scss/components/alert.scss#L31) where we have a similar action. Then we can [simply use the icon in the component structure](https://github.com/SAP/techne/blob/v2.0/test/templates/alert/component.njk#L10). this way, we are repurposing the existing components rather than creating a new one.

Reply:
@saadmhybris I simply re-used what we already have in place: http://sap.github.io/techne/input-group.html (Input Group — Special addon) why not exchanging it with new one everywhere when it's available since alerts example looks totally different from design prospect of view and why would we need different implementations for same button?

Reply:
@saadmhybris as per @xak suggestion we shouldn't care much about the content of the action, it could be delete/edit/sort etc. there even could be few of them. So I just created a container .tn-list-group__button which position any button int he right place. For the delete button I simply re-used .tn-alert__close, it's nothing for the list-group component itself. I also provided a form-related container tn-list-group__action-form which would be useful for checkboxes and radio buttons. This time I don't see the possibility simply re-use the form styling since the long label text needs some additional styling for the list-group, a bit different from what the form has (The label text can't go underneath of the checkbox or radio button, it needs to be positioned next to it with some padding)